doc/manual/manual.tex

```diff
@@ -10042,6 +10042,31 @@ \subsubsection{Brittle thrust wedges benchmark}
 \end{figure}
 
 
+\subsection{Setups for teaching}
```

Comment:
If you put this .tex into a folder like ./cookbooks/teaching/intro-to-geophy/doc/, one could also add ./cookbooks/teraching/intro-to-geophy/README.md that you can point students to (with a simple ordered list like below).

Reply:
I know I probably wouldn't use a README like that (I think I may even just copy the relevant cookbooks to a separate git repo just for a class, in order to be able to add files easily; what I add to the manual is more meant for the instructors rather than the students). If you insist (or if others think it would be useful), I can add a readme, but for now I've simply linked the different cookbook folders in the manual, in addition to the section names.